karma.conf.js

```diff
@@ -0,0 +1,65 @@
+let webpackConfig = require('./webpack.config.js');
+//Need to remove externals otherwise they won't be included in test
+delete webpackConfig.externals;
+// Need to define inline source maps when using karma
+webpackConfig.devtool = 'inline-source-map';
```

Comment:
Use the new convention from other repos for webpack config

Reply:
https://github.com/kaltura/playkit-js/pull/479/

Reply:
done